includes/reader-revenue/woocommerce/class-woocommerce-cover-fees.php

```diff
@@ -112,6 +132,26 @@ private static function should_allow_covering_fees() {
 		return false;
```

Comment:
Non-blocking suggestion: I actually tested changing this to `true` with Woo's standard non-modal checkout flow and it worked flawlessly in that flow, as well. I think we should consider turning it on by default for non-modal donation checkouts too since it works. It would only have any effect for publishers who intentionally use standard checkouts for donations, which could be nobody, but why not? 🤷

Reply:
I guess it would be fine, but the messaging around the feature is all about "donors" and "donations". Let's discover whether it makes sense to ask a customer to pay for the transaction fee when purchasing a product or subscription.

Reply:
It would still only apply to donation transactions because of [this check](https://github.com/Automattic/newspack-plugin/pull/2820/files/9157eba4d68e436c6ce8244f9c1378671734ac68#diff-2f1147d3b8e30d61c3f9c27067874236a35d0aaadee05a80893cda118580d4e5R111). But it's an extreme edge case because donation products are not revealed in the standard Woo shop pages unless the publisher explicitly edits those products to allow it.

Reply:
Whoops, sorry about that. I misinterpreted your inline comment. I agree we can make it a more generic feature

Reply:
Updated in 67bf801d7d629fb094eb5d4209d67601b8d38b03 to support any checkout